src/routers/api/v1/subscriptions.ts

```diff
@@ -0,0 +1,41 @@
+import { APIRouter } from "../router-class"
+import koaBody = require("koa-body")
+import $ = require("transform-ts")
+import { $length } from "../../../utils/transformers"
+import { getRepository } from "typeorm"
+import { Subscription } from "../../../db/entities/subscription"
+import { WP_OPTIONS } from "../../../config"
+
+const router = new APIRouter()
+
+router.get("/", koaBody(), async ctx => {
```

Comment:
getにはbodyがないのでbodyを処理する必要はない
```suggestion
router.get("/", async ctx => {
```